src/selectors/Form/types.ts

```diff
@@ -0,0 +1,25 @@
+import { FormInstance } from '../../instances/Form'
+
+/**
+ * A Selector that provide all {@link FormSelector} methods and adds support
+ * for creating {@apilink DynamicFormInstance | DynamicFormInstances} using
+ * the dynamic syntax.
```

Comment:
This is the first SDK selector that supports the dynamic syntax, but doesn't have any additional methods. With that in mind, we should rephrase this:

```suggestion
 * A Selector that provide a way to create {@apilink FormInstance | FormInstances} 
 * using the dynamic syntax.
```